api/py/ai/chronon/utils.py

```diff
@@ -32,6 +33,23 @@
 chronon_root_path = ""  # passed from compile.py
 
 
+@dataclass
+class Modes:
+    backfill = "backfill"
+    upload = "upload"
+    streaming = "streaming"
+    stats_summary = "stats-summary"
+    label_join = "label-join"
+    log_flattener = "log-flattener"
+    consistency_metrics_compute = "consistency-metrics-compute"
+
+
+@dataclass
+class SubStage:
+    bootstrap = "bootstrap"
+    join_parts = "join_parts"
+
```

Comment:
can we add backfill-left and the corresponding left table?

Reply:
@yuli-han can you help me understand when a backfill-left table is materialized?